core/src/main/java/org/elasticsearch/plugins/MapperPlugin.java

```diff
@@ -52,4 +53,18 @@
     default Map<String, MetadataFieldMapper.TypeParser> getMetadataMappers() {
         return Collections.emptyMap();
     }
+
+    /**
+     * Returns a predicate that fields must match in order to be returned by get mappings, get index and get field mappings API.
+     * Useful to filter the fields that such API return. The predicate receives the index name and the field name as input arguments.
+     */
+    default BiPredicate<String, String> getFieldFilter() {
+        return NOOP_FIELD_FILTER;
+    }
+
+    /**
+     * The default predicate applied, which returns true for any field in any index. That means that by default get mappings, get index
```

Comment:
Maybe `The default predicate for {@link #getFieldFilter} which doesn't filter anything.`?